.github/workflows/ci.yml

```diff
@@ -1,60 +1,10 @@
-name: Make dist and run tests
+name: CI Tests (PRs)
 
 on:
-  push:
-    branches: [master]
   pull_request:
-    branches: [master]
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: [3.12]
-
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ matrix.python-version }}
-
-      - name: Cache pip
-        uses: actions/cache@v4
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-
-
-      - name: Install dependencies
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y libcurl4-openssl-dev
```

Comment:
Wasn't this needed? Or did you remove it because pip is not upgraded now?

Reply:
Nope, I think I just copied and pasted these in thinking they were needed. Seems to run fine without though!